pom.xml

```diff
@@ -148,7 +148,7 @@
     <vers.hbase2>0.98.3-hadoop2</vers.hbase2>
     <vers.hbase-cdh4>0.94.6-cdh${cdh4.version}</vers.hbase-cdh4>
     <vers.hbase-cdh5>0.96.1.1-cdh${cdh5.version}</vers.hbase-cdh5>
-    <vers.hive>0.10.0</vers.hive>
+    <vers.hive>0.13.0</vers.hive>
```

Comment:
CDH4 didn't bundle Kite, so we'd have to push users to use old versions. CDH5 does bundle Kite, so it makes sense to push compatibility concerns to the CDH maintainers.